stock_cycle_count/models/stock_location.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-18 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, fields, models, tools
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from datetime import datetime
+_logger = logging.getLogger(__name__)
+
+try:
+    from numpy import mean
+    NUMPY_PATH = tools.find_in_path('numpy')
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    @api.multi
+    def _compute_loc_accuracy(self):
+        for rec in self:
+            history = self.env['stock.inventory'].search([
```

Comment:
Maybe it could be interesting to have an inventory history field on locations ?

Reply:
what kind of field, could you please explain a little bit more?

Reply:
An 'inventory_history_ids' that represent inventories done on a location.